client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -10,33 +10,31 @@ import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from './purchase-detail';
 
 const BusinessPlanDetails = ( { isFreeTrial, selectedSite } ) => {
-	const showGetFreeDomainTip = ! isFreeTrial;
-
 	return (
 		<div>
-			{ showGetFreeDomainTip
+			{ isFreeTrial
 			? <PurchaseDetail
-					additionalClass="get-free-domain"
-					title={ i18n.translate( 'Get your custom domain' ) }
-					description={
-						i18n.translate(
-							"Replace your site's address, {{em}}%(siteDomain)s{{/em}}, with a custom domain. " +
-							'A free domain is included with your plan.',
-							{
-								args: { siteDomain: selectedSite.domain },
-								components: { em: <em /> }
-							}
-						)
-					}
-					buttonText={ i18n.translate( 'Claim your free domain' ) }
-					href={ '/domains/add/' + selectedSite.slug } />
+				additionalClass="live-chat"
+				title={ i18n.translate( 'Start a Live Chat' ) }
+				description={ i18n.translate( 'Have a question? Chat live with WordPress.com Happiness Engineers.' ) }
+				buttonText={ i18n.translate( 'Talk to an Operator' ) }
+				href="//support.wordpress.com/live-chat/"
```

Comment:
Are there any cases where this shouldn't be https? Will `//` work from inside the desktop app?